recipes/mne-python/construct.yaml

```diff
@@ -86,25 +86,25 @@ specs:
   - mne-rsa =0.9
   - mne-ari =0.1.2
   - mne-kit-gui =1.1.0
-  - mne-icalabel =0.4  # [not win]
+  - mne-icalabel =0.5.1
   - mne-gui-addons =0.1.0
   - mne-lsl =1.0.0
   - traitsui =8.0.0
   - autoreject =0.4.2
   - pyriemann =0.5
-  - pyprep =0.4.2
+  - pyprep =0.4.3
   - openmeeg =2.5.6
   # BLAS
   - openblas =0.3.24
   - libblas =3.9.0=*openblas
   - jupyter =1.0.0
   - jupyterlab =4.0.7
-  - ipykernel =6.25.2
+  - ipykernel =6.26.0
   - nb_conda_kernels =2.3.1
   - spyder-kernels =2.4.4
   - spyder =5.4.5
   - darkdetect =0.8.0
-  - qdarkstyle =3.1
+  - qdarkstyle =3.2
```

Comment:
This seems to be incompatible with Spyder

Reply:
Thanks, I was a bit discourage by the length of the traceback:

```
Could not solve for environment specs
The following packages are incompatible
├─ fsleyes 1.9.0**  is installable with the potential options
│  ├─ fsleyes 1.9.0 would require
│  │  └─ __linux, which is missing on the system;
│  ├─ fsleyes 1.9.0 would require
....
```

and missed the relevant information above.